packages/docs/src/routes/docs/(qwikcity)/action/index.mdx

```diff
@@ -122,6 +122,63 @@ export default component$(() => {
 
 In the example above, the `addUser` action is triggered when the user clicks the button. The `action.submit()` method returns a `Promise` that resolves when the action is done.
 
+## Actions with Event Handlers
+
+We can use the `onSubmitCompleted$` event handler after an action is executed successfully and returns some data. It is particularly useful when you need to perform additional tasks, such as resetting UI elements or updating the application state, once an action has been completed successfully.
+
+Here's an example of the `onSubmitCompleted$` handler used to edit an item in a EditForm component of a todo app.
+
+```tsx title="src/components/EditForm.tsx"
+import { component$, type Signal, useSignal } from '@builder.io/qwik';
+import { Form } from '@builder.io/qwik-city';
+import { type ListItem, useEditFromListAction } from '../../routes/index';
```

Comment:
I think having the full context, e.g. having ListItem and useEditFromListAction in the same file would be better.

Otherwise, looking good!

Reply:
> I think having the full context, e.g. having ListItem and useEditFromListAction in the same file would be better.
> 
> Otherwise, looking good!

Gotcha. The useEditFromListAction and listItem is about another 25 lines. Should I add only those two parts? Or the entire file with the other actions removed?